### ISSUE TITLE
Change tool size (e.g. brush) as Krita does

### ISSUE THREAD

**Issue by Zughy** [LibreSprite/LibreSprite-proposals#39]
##### Is your feature request related to a problem? Please describe
Moving my hand to the other side of the keyboard to change the size of the brush is annoying and it breaks my flow. This is especially true when using graphic tablets

##### Describe the solution you'd like
Change the tool size by holding down `Shift` (left or right doesn't matter) and dragging the mouse/stylus to the left for decreasing and to the right for increasing. In short, 0:52 https://yewtu.be/watch?v=IneodMMWdZg

##### Describe alternatives you've considered
`¯\_(ツ)_/¯`

##### Additional context
`Shift` is right now the shortcut to draw straight lines. We could move that to `Ctrl`, and remove the current kind-of-useless functionality, which allows people to select a layer according to where they clicked (a thing that can turn into a mess pretty easily if someone clicks by accident without noticing)

**Comment by jjhaggar** [comment]
> and remove the current kind-of-useless functionality, which allows people to select a layer according to where they clicked (a thing that can turn into a mess pretty easily if someone clicks by accident without noticing)

Please don't delete that, I use that feature when I'm deep down into layer hell ^^U 

But if you meant "change the shortcut key, but letting the users decide if they want it like it was before anyway", then I'm fine with that ^^ 

Mmmmm... Maybe instead of changing the "straight lines" from ```Shift``` to ```Ctrl```, shouldn't it be easier/better to use ```Ctrl``` as the shortcut for "changing tool size" directly?

**Comment by Zughy** [comment]
> Please don't delete that

Mmm... I can't think of a possible shortcut right now, what would you suggest?

> Mmmmm... Maybe instead of changing the "straight lines" from Shift to Ctrl, shouldn't it be easier/better to use Ctrl as the shortcut for "changing tool size" directly?

I think having the same shortcut Krita has makes life easier for people using both tools... like me haha

**Comment by therealrobster** [comment]
I just wanted to add that CTRL mouse wheel is the perfect solution in my mind.  Shift is used for drawing straight lines.

**Comment by jjhaggar** [comment]
Yeah, CTRL+mouse wheel is also the perfect shortcut for "changing brush size" in my opinion.

**Comment by therealrobster** [comment]
I think this shows that the tools need custom keyboard shortcuts.  You could have a 'aseprite' preset, a 'krita' preset, a 'photoshop' preset etc also, to get people settled in quickly.

**Comment by JamesTDG** [comment]
It would be nice to switch this between ctrl + scroll or shift + scroll instead; I have a nasty hand twitch due to carpal tunnel which would make doing projects hell if the brush size changed any time I am holding ctrl or shift to do a different command
